CHANGELOG.md

```diff
@@ -5,6 +5,11 @@ All notable changes to this project will be documented in this file.
 The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
 and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
+## 0.9.1
```

Comment:
No specific comments about the version you suggest: just remember to consider this: https://semver.org/#what-should-i-do-if-i-update-my-own-dependencies-without-changing-the-public-api